frontend/src/libs/components/icon/icon.tsx

```diff
@@ -5,12 +5,19 @@ import { iconNameToIcon } from './libs/maps/icon-name-to-icon.map.js';
 type Properties = {
   name: IconName;
   color?: string;
+  width?: string | number;
+  height?: string | number;
```

Comment:
It's better to make these properties `number` only and then convert them to `string` and add `px`